Assignment/area_triangle.py

```diff
@@ -0,0 +1,28 @@
+class AreaTriangle:
+
+    def inputdetails(self):
+            base = float(input("Enter your value of base: "))
+            height = float(input("Enter your value of height: "))
+
```

Comment:
a good enhancement would be to check if the input is a number before converting to float 
OR
have a try catch block here, so that if the user inputs alphabets , special characters or spaces etc (i.e. any thing other than numbers) you request the user to enter proper number

Reply:
done